test_suite/generic/test_generic.py

```diff
@@ -400,6 +400,17 @@ def test_langpacks_conf(self, host):
                 assert not host.file(file_to_check).exists, \
                     f'{file_to_check} should not exist in RHEL-8 and above'
 
+    @pytest.mark.run_on(['>=rhel9.6', 'rhel10'])
+    def test_bootc_installed(self, host):
+        """
+        Verify the system-reinstall-bootc package is installed
+        JIRA: CLOUDX-1267
+        """
+
+        with host.sudo():
+            assert host.package("system-reinstall-bootc").is_installed, \
+                'Expected system-reinstall-bootc package not installed'
```

Comment:
Log messages could potentially be more explicit. E.g. "System-reinstall-bootc package expected to be installed in RHEL >= 9.6, 10.0". I know that the test case is only executed on those versions, but I don't know if this is visible from the log output alone.  WDYT?